apps/ui/src/views/Proposal/Votes.vue

```diff
@@ -247,9 +247,9 @@ watch([sortBy, choiceFilter], () => {
               </div>
             </div>
             <div
-              class="leading-[22px] max-w-[144px] w-[144px] flex flex-col justify-center items-end truncate"
+              class="leading-[22px] max-w-[144px] w-[144px] flex flex-col justify-center text-end truncate"
```

Comment:
Can be same as above
```suggestion
              class="leading-[22px] max-w-[144px] w-[144px] flex flex-col justify-center text-right truncate"
```